lib/server.ex

```diff
@@ -86,10 +86,7 @@ defmodule GymTcpApi.Server do
   end
 
   def write_line(socket, {:ok, text}) do
-    if String.strip(text) === "" do
-    else
-      :gen_tcp.send(socket, text)
-    end
+    :gen_tcp.send(socket, text)
```

Comment:
Can you say something about about the error you get if you use ``String.strip``, the problem is, that without the check we could end up in a infinity loop.